ruma/Cargo.toml

```diff
@@ -19,20 +19,20 @@ rustdoc-args = ["--cfg", "docsrs"]
 assign = "1.1.1"
 js_int = "0.2.0"
 
-ruma-common = { version = "0.5.0", path = "../ruma-common" }
-ruma-identifiers = { version = "0.19.0", path = "../ruma-identifiers", features = ["serde"] }
-ruma-serde = { version = "0.3.1", path = "../ruma-serde" }
-
-ruma-client = { version = "=0.5.0-alpha.2", path = "../ruma-client", optional = true }
-ruma-events = { version = "=0.22.0-alpha.3", path = "../ruma-events", optional = true }
-ruma-signatures = { version = "0.7.0", path = "../ruma-signatures", optional = true }
-
 ruma-api = { version = "=0.17.0-alpha.4", path = "../ruma-api", optional = true }
 ruma-appservice-api = { version = "=0.2.0-alpha.3", path = "../ruma-appservice-api", optional = true }
+
+ruma-client = { version = "=0.5.0-alpha.2", path = "../ruma-client", optional = true }
 ruma-client-api = { version = "=0.10.0-alpha.3", path = "../ruma-client-api", optional = true }
+
+ruma-common = { version = "0.5.0", path = "../ruma-common" }
+ruma-events = { version = "=0.22.0-alpha.3", path = "../ruma-events", optional = true }
 ruma-federation-api = { version = "=0.1.0-alpha.2", path = "../ruma-federation-api", optional = true }
+ruma-identifiers = { version = "0.19.0", path = "../ruma-identifiers", features = ["serde"] }
 ruma-identity-service-api = { version = "=0.1.0-alpha.1", path = "../ruma-identity-service-api", optional = true }
 ruma-push-gateway-api = { version = "=0.1.0-alpha.1", path = "../ruma-push-gateway-api", optional = true }
+ruma-serde = { version = "0.3.1", path = "../ruma-serde" }
+ruma-signatures = { version = "0.7.0", path = "../ruma-signatures", optional = true }
```

Comment:
This doesn't make sense. The entries were grouped, now entries from different groups have been switched around. I want the dependencies sorted inside the group, but definitely not across groups.

Reply:
So I would imagine this is common enough to warrant a new `--grouped` flag or something like this I'll add it to `cargo-sort-ck`.

Reply:
If you want to keep "full" sorting as the default, I would recommend that it should at least remove the grouping entirely then (remove the empty lines between dependencies), the current output is just confusing.